app/src/main/java/com/android/shelfLife/model/user/UserRepository.kt

```diff
@@ -5,16 +5,13 @@ import com.android.shelfLife.model.newhousehold.HouseHold
 import kotlinx.coroutines.flow.StateFlow
 
 interface UserRepository {
-  val isUserLoggedIn: StateFlow<Boolean>
```

Comment:
You have decided that the values "isUserLoggedIn" and "selectedHoushold" no longer are part of the userReoisitory